### ISSUE TITLE
Incorrect documentation for -nthreads

### ISSUE THREAD

**Issue by thijsdhollander** [MRtrix3/mrtrix3#1716]
```
  -nthreads number
     use this number of threads in multi-threaded applications (set to 0 to
     disable multi-threading).
```

However, 0 does not disable multi-threading; rather, it fully unleashes it.

I reckon the behaviour is fine, but the documentation doesn't match it.

**Comment by thijsdhollander** [comment]
Feel free to label as you wish.  I had no other options; this seemed to fit "Create a report to help us improve" best. :man_shrugging:

**Comment by thijsdhollander** [comment]
Ok, so it's a bit more nuanced.  Thanks to [this user/post](https://community.mrtrix.org/t/recommended-tractography-options-for-low-b-value/2780/13) on the forum, I've come to realise the setting of this issue only applies to scripts currently.  That probably changes what I would suggest to be the solution: maybe then, the documentation is ok (for binaries, there is a match between documentation and behaviour currently), but the behaviour for the scripting library currently isn't.  Or you could rethink from scratch of course what you want `-nthreads 0` to be in practice.  All up to you.

**Comment by thijsdhollander** [comment]
Should be fixed by #1717 ; closing this.  Happy to have contributed.
